src/library/simplified/extension/marketingABTests.js

```diff
@@ -1,12 +1,38 @@
-var abTests = utag_data.context.tests;
-var marketingABTests = "";
-
-if (typeof abTests !== "undefined") {
-    for (var abTest=0; abTest<abTests.length; abTest++) {
-            var experimentId = utag_data.context.tests[abTest].experimentId;
-            var bucketValue = utag_data.context.tests[abTest].bucketValue;
-            marketingABTests += experimentId + "." + bucketValue + "|";
-    }
+var ABTestExpVals='';
+var tpid = utag_data['context.site.tpid'];
+var eapid = utag_data['context.site.eapid'];
+var guid = utag_data['guid'];
+var siteName = utag_data['context.site.siteName']
+var experimentIds = utag_data['marketingABTests'];
+var expIds = experimentIds.split(",");
+var counter=expIds.length;
+for(var expId=0; expId<counter; expId++) {
+  (function(getId) {
+   var xhttp = new XMLHttpRequest();
+	xhttp.open('GET', 'https://'+siteName+'/api/bucketing/v1/evaluateExperimentsAndLog?guid='+guid+'&tpid='+tpid+'&eapid='+eapid+'&id='+getId+'', true);
```

Comment:
@wmoussa There may be a possibility of CORS(Cross0riginResourceSharing) issue if we try to get response from expedia.com for trunk/int url.
In production we may not experience this issue, but in lab url's we will get this error in the browser console.
Error desc:  **No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://wwwexpediacom.trunk.sb.karmalab.net' is therefore not allowed access.**
I am attaching the screenshot of this(CORS.png)

This is because, we are trying to call prod abacus API from trunk url which creates CORS issue.
In order to come out from this issue, we have to use the corresponding url's. 
Please suggest me if we are ok in this